x-pack/plugins/security_solution/server/usage/detections/ml_jobs/get_initial_usage.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { MlJobUsage } from './types';
+
+/**
+ * Default ml job usage count
+ */
```

Comment:
I think these comments are a little redundant, but if it's part of your coding style it's fine.